### PR TITLE
note that output of jsonencode is minified

### PR DESCRIPTION
**Pull request by brenthc**
The use of `jsonencode` to minify JSON values is useful, so note that in the docs explicitly. A data source or resource may produce a JSON string as an attribute that is not minified, and if needs to be used elsewhere minified, an expression like `jsonencode(jsondecode(data.some_data_source.json))` will do this.

### CODE REVIEW

File: website/docs/language/functions/jsonencode.html.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ Unicode escape sequences: replacing `<`, `>`, `&`, `U+2028`, and `U+2029` with
 `\u003c`, `\u003e`, `\u0026`, `\u2028`, and `\u2029`. This is to preserve
 compatibility with Terraform 0.11 behavior.
 
+The output of `jsonencode` is a minified representation of the input.
```
Comment (laurapacilio) [inline website/docs/language/functions/jsonencode.html.md:41]:
```suggestion
`jsonencode` outputs a minified representation of the input.
```

**Review by laurapacilio — CHANGES_REQUESTED** [review]
Looks good - thank you for adding! Just one nit. Our link check is failing currently. Here's the output:

_Checking URLs in the following pages:
- website/docs/language/functions/jsonencode.html.md

Results:
=== Found broken links! ===
Fix before merging... or if they're not really broken, explain why.

website/docs/language/functions/jsonencode.html.md
  - Broken link: https://tools.ietf.org/html/rfc7159
    [302 Found] https://tools.ietf.org/html/rfc7159_

I think the actual link for RFC 5159 is: https://datatracker.ietf.org/doc/html/rfc7159. It redirects just fine, but could you update that in this file to see if it fixes the link check? If not, just leave a note on your PR about why the link isn't actually broken, and I'll approve.

Thank you! :)

File: website/docs/language/functions/jsonencode.html.md (hunk 1 above)
Comment (brenthc) [reply website/docs/language/functions/jsonencode.html.md]: Pushed the suggested wording change.

**Comment by laurapacilio** [comment]
ignoring the link checker as it's just flagging a redirect - the link isn't actually broken.

**Pull request merged**